app/javascript/packs/stickySaveFooter.js

```diff
@@ -0,0 +1,7 @@
+const form = document.querySelector('.edit_user');
+
+form.addEventListener('change', () => {
+    const saveFooter = document.querySelector('#save-footer');
+    saveFooter.classList.add('sticky-save-footer');
```

Comment:
to be more robust, always check if the element is there after querying it and know that `getElementByClassName` is much faster than `querySelector`: https://jsperf.com/getelementsbyclassname-0-vs-queryselector/213